### DISCUSSION TITLE
Use Stork-search for offline static website?

### DISCUSSION THREAD

**Discussion by icestars** [jameslittle230/stork#315] [Q&A] [answered]
Sir! How can I use stork-search for offline static website?

**Comment by jameslittle230** [comment, marked as answer]
Hi @icestars! Regrettably, you might not be able to today. You'd run into the same issues [this StackOverflow asker saw](https://stackoverflow.com/questions/61052684/how-to-load-a-wasm-module-locally), where you'd have trouble loading the `stork.wasm` file from a local URI, and even if you did, the browser would refuse to load it because it wasn't being served with the `application/wasm` MIME type.

I'll likely need to change the way the WASM file is loaded in order to fix this - I could probably follow the top answer's suggestion of encoding and embedding the WASM binary within the script file. This would be a major architectural change, and I probably wouldn't release it until I ship Stork 2.0.0.

Sorry that that's likely not the answer you were hoping for. Please let me know if I can be helpful otherwise!

James

**Reply by icestars** [reply]
Thanks for you anwser. I'll wait for Stork 2 and hope it will be coming very soon
